roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -46,18 +46,25 @@ def run(self, tmp=None, task_vars=None):
 
         result["checks"] = check_results = {}
 
+        user_disabled_checks = [
+            check.strip()
+            for check in task_vars.get("openshift_disable_check", "").split(",")
+        ]
+
         for check_name in resolved_checks:
             display.banner("CHECK [{} : {}]".format(check_name, task_vars["ansible_host"]))
             check = known_checks[check_name]
 
-            if check.is_active(task_vars):
+            if check_name not in user_disabled_checks and check.is_active(task_vars):
                 try:
                     r = check.run(tmp, task_vars)
                 except OpenShiftCheckException as e:
                     r = {}
                     r["failed"] = True
                     r["msg"] = str(e)
             else:
+                # TODO(rhcarvalho): we may want to provide some distinctive
+                # complementary message to know why a check was skipped.
```

Comment:
We may want to distinguish why a check was skipped, not necessarily to be implemented in this PR.